addons/im_livechat/views/im_livechat_channel_views.xml

```diff
@@ -204,6 +204,7 @@
                     <field name="sequence" widget="handle"/>
                     <field name="regex_url"/>
                     <field name="action"/>
+                    <field name="chatbot_script_id"/>
```

Comment:
Maybe make a separate PR for this as we could merge this immediately (even still in 18.3)